scripts/common

```diff
@@ -64,3 +64,44 @@ else
     '--remote_upload_local_results=false'
   )
 fi
+
+# kill_pid tries to kill the given pid(s), first softly then more aggressively.
+kill_pid() {
+  local pids=( "$@" )
+  echo "Killing ${pids[*]}"
+  set +e
+  local count=0
+  while kill -INT "${pids[@]}" > /dev/null 2>&1; do
+    sleep 1
+    ((count++))
+    still_alive=()
+    for pid in "${pids[@]}"; do
+      if ps -p "${pid}" > /dev/null ; then
+        still_alive+=("${pid}")
+      fi
+    done
+    if [[ ${#still_alive[@]} -eq 0 ]]; then
+      break
+    fi
+    if [[ $count -gt 5 ]]; then
```

Comment:
Why is it 5?

Reply:
No reason, just a guess to give the process ~5 seconds to shut down gracefully.